modules/users/server/tests/user.controller.spec.js

```diff
@@ -0,0 +1,576 @@
+process.env.NODE_ENV = 'test';
```

Comment:
Not really rework, just wanted to be sure this wasn't generated code being committed.  It kinda looks like a test though... just confirm and I'll approve.  Everything else looks good.